content/docs/1.1.1/deploy/install/_index.md

```diff
@@ -106,6 +106,14 @@ For RHEL, CentOS, and EKS with `EKS Kubernetes Worker AMI with AmazonLinux2 imag
 
 ```
 yum install iscsi-initiator-utils
+echo "InitiatorName=$(/sbin/iscsi-iname)" > /etc/iscsi/initiatorname.iscsi
```

Comment:
The above command is not right, and take this time to update as below as well.

`yum --setopt=tsflags=noscripts install iscsi-initiator-utils`

Reply:
Also, some words need to change as well.

`For RHEL, CentOS, and EKS with EKS Kubernetes Worker AMI with AmazonLinux2 image, use below commands:`

Reply:
Modified. Thanks.